content/courses/native-onchain-development/paging-ordering-filtering-data-frontend.md

```diff
@@ -147,21 +180,32 @@ For example, you could search through a list of contacts by including a `memcmp`
 filter:
 
 ```tsx
+const DATA_OFFSET = 2; // The first 2 bytes of metadata they store versioning information and are not needed for the search.
+const DATA_LENGTH = 18; // Retrieve 18 bytes of relevant data - This length includes the specific part of the account's data that holds the necessary information for comparison.
```

Comment:
What information specifically? Be specific. 

It's probably "use the first 18 bytes of data to compare the contacts", just taking a guess from the context.

Reply:
fixed ✅